docs/pages/database-access/guides/redis.mdx

```diff
@@ -33,7 +33,15 @@ This guide will help you to:
 
 - Teleport version `9.0` or newer.
 - Redis version `6.0` or newer.
-- `redis-cli` installed and added to your system's `PATH` environment variable.
+
+- `redis-cli` version `6.2` or newer installed and added to your system's `PATH` environment variable.
```

Comment:
Approve, but with a nit. Usually I prefer to use version variables even if the version rarely changes, e.g. `min_redis_version: 9.0` so it's easier to track all references to it in the future and update if the requirements change.

Reply:
👍 I've created a docs issue so we can move all hardcoded versions into variables in one go: https://github.com/gravitational/teleport/issues/14864